tools/kdc.json

```diff
@@ -6,30 +6,24 @@
   "container": {
     "type": "MESOS",
     "docker": {
-      "image": "mesosphere/kdc:latest",
+      "image": "nvaziri/kdc:dev",
```

Comment:
@NimaVaziri where does the Dockerfile for this live?

Reply:
I included it in `tools/` (perhaps there's a better place), but it's literally just a `RUN` instruction that invokes a shell script.